src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { Todo } from '../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList : React.FC <Props> = ({ todos }) => (
+  <section>
+    {todos.map(todo => (
+      <TodoInfo
+        key={todo.id}
+        todo={todo}
+      />
+    ))}
+  </section>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings